configure.ac

```diff
@@ -2420,8 +2420,7 @@ dnl AC_MSG_RESULT($cpp_type)
 
 # checks for header files
 AC_HEADER_STDC
-AC_CHECK_HEADERS(asm/types.h crypt.h conio.h direct.h dlfcn.h errno.h \
```

Comment:
What's this about?

Reply:
I will remove them

Reply:
Hmm, this failed the build

Reply:
huh?

Reply:
should I revert https://github.com/msys2-contrib/cpython-mingw/pull/2/commits/c9a1db36c179accafd5fdb4e27cdb1f77aecef61 ?

Reply:
yeah, might be due to missing quoting maybe? We can look later

Reply:
but ideally the autotools change would be a separate commit.

Reply:
Is it because of line ending?